stack_definition.json

```diff
@@ -717,35 +717,35 @@
   "outputs": [
     {
       "name": "elasticsearch_hostname",
-      "value": "ref:./members/3b - Databases for Elasticsearch/outputs/hostname"
+      "value": "ref:../3b - Databases for Elasticsearch/outputs/hostname"
     },
     {
       "name": "elasticsearch_port",
-      "value": "ref:./members/3b - Databases for Elasticsearch/outputs/port"
+      "value": " ref:../3b - Databases for Elasticsearch/outputs/port"
```

Comment:
@akocbek - I think you have an extra leading space

Reply:
@in-1911 good catch, thanks